config-model/src/main/perl/vespa-deploy

```diff
@@ -148,29 +144,16 @@ if ($opt_p) {
   $port = $opt_p;
 }
 
-if ($opt_z) {
-  $cert = $opt_z;
-}
-
 $pathPrefix = "/application/v2/tenant/$tenant/session";
 
-$siaPath = "/var/lib/sia/";
-
-$siaCertsPath = $siaPath . "certs/";
-
-$siaKeysPath = $siaPath . "keys/";
-
 create_cloudconfig_dir();
 
 $session_id_file = "$cloudconfig_dir/$tenant/deploy-session-id";
 
 my $command = shift;
 $command ||= "help";
 
-my $curl_command = 'curl -A vespa-deploy --silent --show-error --connect-timeout 30 --max-time 1200';
-if ($cert) {
-    $curl_command = $curl_command . " -k --cert " . $siaCertsPath .  $cert . ".cert.pem --key " . $siaKeysPath . $cert . ".key.pem ";
-}
+my $curl_command = '$VESPA_HOME/libexec/vespa/vespa-curl-wrapper -A vespa-deploy --silent --show-error --connect-timeout 30 --max-time 1200';
```

Comment:
Is it on purpose that $VESPA_HOME is not interpolated?

Reply:
Nope. Good catch 👍